kod/object/item/passitem/spelitem.kod

```diff
@@ -193,7 +193,8 @@ messages:
    }
    
    CastSpell(what = $, apply_on = $)
-   "Casts the spell set by the spell object"
+   "Casts the spell set by the spell object, returning TRUE\n"
+   "If successful, or FALSE if the spell failed for whatever reason."
```

Comment:
formatting is weird here: I is capital and there's no space before it.

Reply:
My bad, all fixed.